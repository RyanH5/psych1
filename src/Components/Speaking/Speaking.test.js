import React from 'react';
import { shallow } from 'enzyme';
import { Speaking } from './speaking';


describe('Speaking', () => {
  it('should match snapshot', () => {
    let wrapper = shallow(<Speaking />)

    expect(wrapper).toMatchSnapshot();
  });
});