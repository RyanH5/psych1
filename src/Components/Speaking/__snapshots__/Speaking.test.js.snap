// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Speaking should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Speaking and Training
</h1>,
        <p>
          I have been involved in speaking and providing resilience training to schools, mental health professionals, church and youth workers, and small businesses and agencies for over 25 years. The speaking and training focuses on developing specific mind-body skills that professionals can use to help people with their physical health, brain health and emotional health.These skills help people utilize their internal resources to make powerful changes in their healing and growth.
</p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Speaking and Training",
        },
        "ref": null,
        "rendered": "Speaking and Training",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "I have been involved in speaking and providing resilience training to schools, mental health professionals, church and youth workers, and small businesses and agencies for over 25 years. The speaking and training focuses on developing specific mind-body skills that professionals can use to help people with their physical health, brain health and emotional health.These skills help people utilize their internal resources to make powerful changes in their healing and growth.",
        },
        "ref": null,
        "rendered": "I have been involved in speaking and providing resilience training to schools, mental health professionals, church and youth workers, and small businesses and agencies for over 25 years. The speaking and training focuses on developing specific mind-body skills that professionals can use to help people with their physical health, brain health and emotional health.These skills help people utilize their internal resources to make powerful changes in their healing and growth.",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Speaking and Training
</h1>,
          <p>
            I have been involved in speaking and providing resilience training to schools, mental health professionals, church and youth workers, and small businesses and agencies for over 25 years. The speaking and training focuses on developing specific mind-body skills that professionals can use to help people with their physical health, brain health and emotional health.These skills help people utilize their internal resources to make powerful changes in their healing and growth.
</p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Speaking and Training",
          },
          "ref": null,
          "rendered": "Speaking and Training",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "I have been involved in speaking and providing resilience training to schools, mental health professionals, church and youth workers, and small businesses and agencies for over 25 years. The speaking and training focuses on developing specific mind-body skills that professionals can use to help people with their physical health, brain health and emotional health.These skills help people utilize their internal resources to make powerful changes in their healing and growth.",
          },
          "ref": null,
          "rendered": "I have been involved in speaking and providing resilience training to schools, mental health professionals, church and youth workers, and small businesses and agencies for over 25 years. The speaking and training focuses on developing specific mind-body skills that professionals can use to help people with their physical health, brain health and emotional health.These skills help people utilize their internal resources to make powerful changes in their healing and growth.",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
